docs/standard/serverless-architecture/index.md

```diff
@@ -0,0 +1,127 @@
+
+All rights reserved. No part of the contents of this book may be reproduced or transmitted in any form or by any means without the written permission of the publisher.
+
+This book is provided “as-is” and expresses the author’s views and opinions. The views, opinions and information expressed in this book, including URL and other Internet website references, may change without notice.
+
+Some examples depicted herein are provided for illustration only and are fictitious. No real association or connection is intended or should be inferred.
+
+Microsoft and the trademarks listed at http://www.microsoft.com on the “Trademarks” webpage are trademarks of the Microsoft group of companies.
+
+Mac and macOS are trademarks of Apple Inc.
+
+The Docker whale logo is a registered trademark of Docker, Inc. Used by permission.
+
+All other marks and logos are property of their respective owners.
+
+Author:
+
+> **Jeremy Likness**, Sr. Cloud Developer Advocate, APEX, Microsoft Corp.
+
+Editors:
+
+Participants and reviewers:
+
+## Introduction
+
+Serverless is the evolution of cloud platforms in the direction of pure cloud native code that brings developers closer to business logic while insulating them from infrastructure concerns. It’s a pattern that doesn’t imply “no server” but rather, “less server”. Serverless code is event-driven. Code may be triggered by anything from a traditional HTTP web request to a timer or the result of uploading a file. The infrastructure behind serverless allows for instant scale to meet elastic demands and offers micro-billing to truly “pay for what you use.” Serverless requires a new way thinking and approach to building applications and is not the right solution for every problem. As a developer, you must decide:
+
+* What are the pros and cons of serverless?
+* Why should you consider serverless for your own applications?
+* How can you build, test, deploy, and maintain your serverless code?
+* Where does it make sense to migrate code to serverless in existing applications, and what is the best way to accomplish transformation?
+
+## About this guide
+
+This guide focuses on cloud native development of applications that use serverless. It highlights the benefits and exposes the potential drawbacks of developing serverless apps and provides a survey of serverless architectures. It illustrates numerous examples of how serverless can be used and details various serverless design patterns.
+
+This guide explains the components of the Azure serverless platform and focuses specifically on implementation of serverless using [Azure Functions](https://docs.microsoft.com/azure/azure-functions/functions-overview). You will learn about triggers and bindings as well as how to implement serverless apps that rely on state using durable functions. Finally, business examples and case studies will help provide context and a frame of reference to determine whether serverless is the right approach for your projects.
```

Comment:
This could be just me, but as you introduce Azure Functions, I'd like to know if there is any relation to the concept of a "Pure Function", (in the functional programming sense), or is it just some code that runs when needed....

Reply:
will cover that in the Azure Functions chapter if that makes sense

Reply:
Testing a checklist in my own comment: 

[ ] Todo: try to check this

Reply:
Try again with this: 

- [ ] Todo: try to check this
- [ ] And this

Reply:
It works with my own comments.